.ci_support/linux_64_c_compiler_version10cuda_compiler_version11.2cxx_compiler_version10python3.10.____cpython.yaml

```diff
@@ -23,15 +23,15 @@ docker_image:
 giflib:
 - '5.2'
 grpc_cpp:
-- '1.51'
+- '1.52'
 icu:
 - '72'
 libabseil:
-- '20220623.0'
+- '20230125'
 libcurl:
-- '7'
+- '8'
 libgrpc:
-- '1.51'
+- '1.52'
```

Comment:
Would it be realistic to bump to grpc 1.54 here? That would be the "current" one before the (likely bigger change) protobuf 4

Reply:
As mentioned in other threads, I would prefer to build both variants, 1.51 and 1.52.

Reply:
> As mentioned in other threads, I would prefer to build both variants, 1.51 and 1.52.

Nothing supports 1.51 anymore. We [closed](https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4370) the 1.52 migration 2 ago, and the 1.54 migration has effectively been done since that time as well (only missing tensorflow and a package that doesn't build anymore). In other words, going to 1.52 wouldn't help much with conflicts against other current packages.

In case that was perhaps confusing with the protobuf 4 situation, I'm not proposing to move to grpc >= 1.55 (which would require protobuf 4).

Reply:
I was talking about 1.54 and 1.52 instead of 1.51 🙈 . I can then bump later to 1.54 and see whether it still passes.

Reply:
OK cool, glad we double-checked. ;-)

To conclude my point though: Everything but tensorflow else has moved to 1.54. I would normally say "go ahead" if you want to build it twice also for 1.52 just to make sure, but considering the human (& compute) resources necessary to pull off a new version here, I would strongly suggest to skip 1.52.

Reply:
Somewhat related note - I think the libgrpc migrator should have migrated grpc_cpp as well. At the moment they are not in sync, but they should: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/746eb797a786a605e6b5d0754dfcbbc27beba510/recipe/conda_build_config.yaml#L396-L398

Reply:
No, it's unrelated. `grpc_cpp` was removed for 1.52, see [here](https://github.com/conda-forge/grpc-cpp-feedstock/issues/291). Where you're right is that the comment in the global pinning is now wrong, I'll go fix that.